lib/govuk_index/publishing_event_worker.rb

```diff
@@ -50,7 +50,7 @@ def process_action(actions, routing_key, payload)
       if presenter.unpublishing_type?
         logger.info("#{routing_key} -> DELETE #{presenter.base_path} #{presenter.type}")
         actions.delete(presenter)
-      elsif MigratedFormats.indexable?(presenter.format) && payload['publishing_app'] != 'smartanswers'
+      elsif MigratedFormats.indexable?(presenter.format)
```

Comment:
Does this mean we can get rid of all the `regenerate_if: ->(example) { example["publishing_app"] == "smartanswers" }` code? 😍

Reply:
potentailly, but I haven;t tested it and would prefer to do that in a separate PR.